ts/Extensions/OverlappingDataLabels.ts

```diff
@@ -199,7 +199,8 @@ Chart.prototype.hideOverlappingLabels = function (
                 }
 
                 return {
-                    x: pos.x + (parent.translateX || 0) + padding - xOffset,
+                    x: pos.x + (parent.translateX || 0) + padding -
+                        (xOffset || 0),
```

Comment:
Is this for `xOffset === NaN`? It is initialized to 0 above it seems

Reply:
Yes but there are some cases where `alignValue`, `textAlign` and `translateX` properties can be undefined therefore the expression in the `else if` section can result in `NaN`.